### PR TITLE
Fix/character input field fix for small screens and iPad

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Fix the Code entering screen layout issue on iPad and small screen iPhones.
Decrease CharacterView's width on iPhone with 320 pt width.
Remove redundant constraints.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Registration/RegistrationTeam/Flow/TeamCreationStepController.swift [hunk 1]
```diff
@@ -101,7 +101,7 @@ final class TeamCreationStepController: UIViewController {
     func updateKeyboardOffset(keyboardHeight: CGFloat) {
         self.keyboardOffset.constant = -(keyboardHeight + 10)
         UIView.performWithoutAnimation {
-            self.view.layoutIfNeeded()
+            self.view.setNeedsLayout()
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Registration/RegistrationTeam/Flow/TeamCreationStepController.swift]: What is the intention here? To make view layout you need to do both: setNeedsLayout and immediately after layoutIfNeeded.

File: Wire-iOS/Sources/UserInterface/Components/CharacterInputField.swift [hunk 2]
```diff
@@ -172,10 +177,12 @@ public class CharacterInputField: UIControl, UITextInputTraits {
     
     // MARK: - Overrides
     
-    init(maxLength: Int, characterSet: CharacterSet) {
+    init(maxLength: Int, characterSet: CharacterSet, width: CGFloat) {
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Components/CharacterInputField.swift]: `width` is not descriptive. Also if we would like to change the height too then we'll need to introduce the new parameter. Please consider replacing with `charPlaceholderSize: CGSize`.

File: Wire-iOS/Sources/UserInterface/Registration/RegistrationTeam/Flow/TeamCreationStepController.swift (hunk 1 above)
Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Registration/RegistrationTeam/Flow/TeamCreationStepController.swift]: will undo the change

**Comment by mikeger** [comment]
@billypchan I tried merging develop into this PR but failed merging `TeamCreationStepController.swift`. Please check if you can do it.

**Pull request merged**